### DISCUSSION TITLE
Teria o Gemidão do Zap ido longe demais?

### DISCUSSION THREAD

**Discussion by haskellcamargo** [frontendbr/forum#648] [Dicas]
Eu e meus amigos escrevemos isso:

https://github.com/haskellcamargo/gemidao-do-zap

É uma aplicação em NodeJS de linha de comando que, consumindo a API da TotalVoice, liga para o número que você escolher e toca o clássico gemidão do zap.
Estamos agora começando a escrever o front em React. Achei que seria útil apresentar o projeto a vocês. :smile: 

Be happy!

**Comment by lucianobragaweb** [comment]
Doideira esse projeto!!! Parabéns!!

**Comment by raulfernando** [comment]
Testei aqui funcionou hahahaha muito bom 

Obs: vi uma galera no face dizendo que não conseguiram os créditos aqui foi de boa ganhei 3 reais.

**Comment by kvnol** [comment]
HAHAHAHAHAHAHAHAHAHAHAHAHAH, vocês são demais, pqp.

**Comment by felipefialho** [comment]
hahahaha

**Comment by rubenmarcus** [comment]
genial!!! ja mandei pra minha lista do whats toda!!!

**Comment by jvmaia** [comment]
Posso dar um PR com o cliente que eu fiz para um bot de telegram? Só precisa deixar o script rodando que seu bot tá funcionando, e você pode rodar o comando de onde estiver.

**Comment by gabrielhsp** [comment]
Sensacional hahahahaha

**Comment by anabastos** [comment]
@haskellcamargo Está em terceiro no trending repositories e trending developers global do github. Parabéns cellinho vc é top :open_mouth: :tada: :clap:

**Comment by gabrieldarezzo** [comment]
Vocês viram neh?

https://olhardigital.com.br/noticia/brasileiros-criam-ferramentas-que-enviam-o-gemidao-por-chamada-telefonica/70373

HSUahsuashuHAsA

Edit:
Detalhe no Who is:

Name:Cid Nao Salvo
Organization:Nao Salvo

**Comment by nandomoreirame** [comment]
Eu acabei de fazer o **Lorem {gemidão do zap} ipsum**

https://github.com/nandomoreirame/lorem-gemidao-do-zap-ipsum

**Comment by WandersonAlves** [comment]
Os designers piram!!! @nandomoreirame
